### PR TITLE
fix(UnifiedSearch): Restore normal dialog size

### PR DESCRIPTION
**Pull request by nfebe**
| Before | After |
| --------- | ------ |
| Screenshot from 2024-08-30 10-10-14 | Screenshot from 2024-08-30 10-07-34 |

### CODE REVIEW

**Review by sorbaugh — APPROVED** [review]

**Review by marcoambrosini — APPROVED** [review]

File: core/src/components/UnifiedSearch/UnifiedSearchModal.vue [hunk 1]
```diff
@@ -724,7 +725,8 @@ export default defineComponent({
 	&__no-content {
 		display: flex;
 		align-items: center;
-		height: 100%;
+		margin-top: 0.5em;
```
Comment (susnux) [inline core/src/components/UnifiedSearch/UnifiedSearchModal.vue:728]: What is that margin for? Should be already covered in the height, no?

**Review by susnux — APPROVED** [review]
Makes sense

File: core/src/components/UnifiedSearch/UnifiedSearchModal.vue (hunk 1 above)
Comment (Fenn-CS) [reply core/src/components/UnifiedSearch/UnifiedSearchModal.vue:728]: The borders are overlapping with elements above....

Comment (susnux) [reply core/src/components/UnifiedSearch/UnifiedSearchModal.vue:728]: ok thank you :)

**Review by jancborchardt — APPROVED** [review]
Good stuff, thanks @Fenn-CS!

**Pull request merged**